modules/gitops-argocd-cli-creating-an-application-in-normal-mode.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * argocd_applications/creating-an-application-using-gitops-argocd-cli.adoc
+// * declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+ifeval::["{context}" == "configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations"]
+:cluster:
+endif::[]
+ifeval::["{context}" == "creating-an-application-using-gitops-argocd-cli"]
+:app:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argocd-cli-creating-an-application-in-normal-mode_{context}"]
+= Creating an application in normal mode by using the GitOps CLI
+
+You can create applications in normal mode by using the {gitops-shortname} `argocd` CLI.
+
+ifdef::cluster[]
+This sample workflow walks you through the process of configuring Argo CD to recursively sync the content of the `cluster` directory to the `cluster-configs` application. The directory defines the {OCP} cluster configurations and a namespace `spring-petclinic` on the cluster.
+endif::cluster[]
+
+.Prerequisites
+
+* You have installed the {gitops-title} Operator in your cluster.
+* You have logged into Argo CD instance.
```

Comment:
Doesn't this mean you already have argocd CLI in place? IMO 4th prereq should be bumped up